topics/create-first-app.md

```diff
@@ -6,7 +6,7 @@ Here you will learn how to create and run your first KMM application.
 1. [Set up your environment for KMM development](setup.md) by installing the necessary tools on a suitable operating system.  
     
     > To work with shared code or Android-specific code, use any computer with an operating system supported by [Android Studio](https://developer.android.com/studio).  
```

Comment:
I've read this warning and it's not correct in this context. You need a mac with macOS for this tutorial, so there is no way.
I would use the text from the new tutorial:
> You will need a Mac with macOS to complete certain steps in this tutorial, which include writing iOS-specific code and running an iOS application.
> These steps cannot be performed on other operating systems, such as Microsoft Windows. This is due to an Apple requirement.

Also copyeditors are now removing an article before macOS. Please remove it in other places as well.